### PR TITLE
[CLOUDTRUST-2438] Remove parameter groupIDs from GET /kyc/users

### PR DESCRIPTION
**Pull request by fperot74**

### CODE REVIEW

File: pkg/kyc/authorization.go [hunk 1]
```diff
@@ -68,17 +68,15 @@ func (c *authorizationComponentMW) GetActions(ctx context.Context) ([]apikyc.Act
 	return c.next.GetActions(ctx)
 }
 
-func (c *authorizationComponentMW) GetUserByUsername(ctx context.Context, username string, groupIDs []string) (apikyc.UserRepresentation, error) {
+func (c *authorizationComponentMW) GetUserByUsername(ctx context.Context, username string) (apikyc.UserRepresentation, error) {
 	var action = KYCGetUserByUsername.String()
 	var targetRealm = c.realmName
```
Comment (harture) [inline pkg/kyc/authorization.go]: We are using realmName of the MW coming from configuration. I think it should depend of the current user and thus using the value present in context. Am I missing something ?

Comment (fperot74) [reply pkg/kyc/authorization.go]: Right... fixed

Comment (harture) [reply pkg/kyc/authorization.go]: This issue seems to also applies to the others methods

Comment (fperot74) [reply pkg/kyc/authorization.go]: Re-fixed

**Pull request merged**